app/jobs/upload_uphold_access_parameters_job.rb

```diff
@@ -3,7 +3,7 @@ class UploadUpholdAccessParametersJob < ApplicationJob
 
   def perform(publisher_id:)
     publisher = Publisher.find(publisher_id)
-
+    return if publisher.uphold_connection.uphold_verified
```

Comment:
My only concern is that people who want to go and reconnect their uphold after they've revoked access